app/routes/sources/new.js

```diff
@@ -1,18 +1,24 @@
 import Route from '@ember/routing/route';
 import SourceSnapshot from '../../utils/source-snapshot';
 import CONFIG from '../../config/constants';
+import { inject as service } from '@ember/service';
 
 export default class SourcesNewRoute extends Route {
+  @service session;
+
   async model() {
+    const creator = await this.session.currentSession.organization;
```

Comment:
Je kan onmiddellijk de `currentSession` service injecten i.p.v. `this.session.currentSession` te doen.

Aangezien de `load()` method van de `currentSession` service ge-await wordt [in de `beforeModel`-hook van de application route](https://github.com/kanselarij-vlaanderen/frontend-persberichten/blob/master/app/routes/application.js#L10) en in die `load()` method [het laden van de `organization`](https://github.com/kanselarij-vlaanderen/frontend-persberichten/blob/bc83e308a0c73564be40796f0cd7b464b362e938/app/services/current-session.js#L24) ge-await wordt, lijkt `currentSession.organization` mij geen promise terug te geven maar een object. Die `await` is dus overbodig volgens mij. 

Deze comments gelden op nog plaatsen (ook in de bestaande code om een creator in te vullen bij het aanmaken van een persbericht).